src/libraries/System.Reflection/tests/ConstructorInfoTests.cs

```diff
@@ -68,6 +68,25 @@ public void Invoke_StaticConstructor_NullObject_NullParameters()
             Assert.Null(obj);
         }
 
+        [Fact]
+        public void Invoke_StaticConstructorMultipleTimes()
+        {
+            ConstructorInfo[] constructors = GetConstructors(typeof(ClassWithStaticConstructorThatIsCalledMultipleTimesViaReflection));
```

Comment:
BTW this test library doesn't run against .NET Framework, so you'd have to paste this into a test program if you wanted to be sure NETFX behavior was the same.